### DISCUSSION TITLE
Cookie consent script

### DISCUSSION THREAD

**Discussion by HuubMons** [HuubMons/HuMo-genealogy#40] [Ideas]
I'm searching for a better Cookie consent script.
There is no official Cookie consent message in HuMo-genealogy at the moment, only a Cookie page.

It's better to replace it with a cookie message.
https://github.com/FriendsOfREDAXO/consent_manager
Or simple cookie consent banner:
https://mountcreo.com/article/creating-a-cookie-consent-banner-from-start-to-finish-in-php/
And other options:
https://www.jqueryscript.net/blog/best-free-gdpr-cookie-consent-banner.html

**Comment by ghost** [comment]
Here the best I think : https://tarteaucitron.io/en/

Your fist link look nice to. your second = never and all in third link are not user friendly.
